assembly-main/src/assembly/bin/che.sh

```diff
@@ -594,22 +620,22 @@ function launch_che_server {
   # Launch Che as a docker image
   else
 
-    echo "Starting Che server in docker container named che."
+    echo -e "Starting Che server in docker container named ${GREEN}${CONTAINER}${NC}."
 
     CREATE_NEW_CONTAINER=false
 
     # Check to see if the Che docker was not properly shut down
-    "${DOCKER}" inspect che &> /dev/null || DOCKER_INSPECT_EXIT=$? || true
+    "${DOCKER}" inspect ${CONTAINER} &> /dev/null || DOCKER_INSPECT_EXIT=$? || true
     if [ "${DOCKER_INSPECT_EXIT}" != "1" ]; then
 
       # Existing container running Che is found.  Let's start it.
-      echo -e "Found a container named ${GREEN}che${NC}. Attempting restart."
-      "${DOCKER}" start che &>/dev/null || DOCKER_EXIT=$? || true
+      echo -e "Found a container named ${GREEN}${CONTAINER}${NC}. Attempting restart."
+      "${DOCKER}" start ${CONTAINER} &>/dev/null || DOCKER_EXIT=$? || true
 
       # Existing container found, but could not start it properly.
       if [ "${DOCKER_EXIT}" == "1" ]; then
         echo "Initial start of docker container failed... Attempting docker restart and exec."
-        "${DOCKER}" exec che bash -c 'true && sudo service docker start && tail -f /dev/null' || DOCKER_EXIT=$? || true   
+        "${DOCKER}" exec ${CONTAINER} bash -c 'true && sudo service docker start && tail -f /dev/null' || DOCKER_EXIT=$? || true   
```

Comment:
Why you are using `true && ` at the beginning of command? In my opinion it does nothing